.github/workflows/lint-markdown.yml

```diff
@@ -16,4 +16,6 @@ jobs:
         uses: docker://avtodev/markdown-lint:v1
         with:
           args: './**/*.md'
+          # check ./.meta/.markdownlintignore
           ignore: './**/CHANGELOG.md ./**/ios/** ./**/android/** ./docs/database/**'
```

Comment:
If we can't use the ignore file with this action I think we should consider just switching to [this more official container](https://github.com/igorshubovych/markdownlint-cli/pkgs/container/markdownlint-cli) for the workflow

Reply:
Should we postpone that to another issue? I don't want to spend time on porting the markdown linter currently ^^

Reply:
Created #477